### ISSUE TITLE
How to use biobjective function in NOMAD.jl

### ISSUE THREAD

**Issue by sivabkumar** [bbopt/NOMAD.jl#36]
In the user guide of NOMAD they said that NOMAD will be able to handle biobjective function also. Is it possible in our NOMAD.jl also?

**Comment by salomonl** [comment]
Infortunately, `NOMAD.jl` is based on the software NOMAD 4, which does not implement a biobjective optimization algorithm. You have to use the `C++` version of NOMAD 3.9 to have this functionality.

**Comment by ufechner7** [comment]
Well, you can always create a resulting objective that is a linear combination of any number of objectives with different weights. I think this will just work. By changing the weights you get the Pareto front...

**Comment by salomonl** [comment]
Not really, with this type of approach, you could miss non-convex parts of the Pareto front. You could reimplement the BiMADS algorithm given in this paper 

> MULTIOBJECTIVE OPTIMIZATION THROUGH A SERIES OF SINGLE-OBJECTIVE FORMULATIONS

or if you need more objectives the MultiMADS algorithm

> A mesh adaptive direct search algorithm for multiobjective optimization

If you have enough patience, note that we are planning to upgrade NOMAD.jl, to make use of a new multiobjective algorithm, DMulti-MADS, implemented in NOMAD 4 but still in beta testing for the moment.
